docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -366,8 +366,56 @@ revisit this and keep track of dad as well as moms).
 Code for pulling GBD ASFR appears in `recent Maternal IV Iron model
 <https://github.com/ihmeuw/vivarium_gates_iv_iron/blob/67bbb175ee42dce4536092d2623ee4d83b15b080/src/vivarium_gates_iv_iron/data/loader.py#L166>`_.
 
-Multiparity --- make twins with probability from ACS, about 2.5%.  See
-Section (16) for additional details.
+Multiparity --- make twins with probability 4%.  See Section (16) for
+additional details.
+
+Relationship -- the sim knows a parent-child dyad when the new
```

Comment:
Does the parent in the dyad represent the parent giving birth? Are there restrictions on sex?